### ISSUE TITLE
Jorginho BOT on Instagram

### ISSUE THREAD

**Issue by gustavo-oliveira-rosa** [Raisess/jorginho-bot-on-instagram#1]
Já pensou em criar uma API ou aumentar esse seu projeto?

**Comment by Raisess** [comment]
Já sim, por enquanto estou trabalhando em alguns projetos privados e resolvi deixar esse um pouco de lado. Mas algum dia ele pode sim virar uma API.

Fique a vontade para contribuir e usar o projeto. <3

**Comment by gustavo-oliveira-rosa** [comment]
Claro! Vou abrir alguns PRS aqui, e se criassemos uma comunidade pra ele?

Vamos ampliar o open-source, também tenho muitos projetos, mas contribuir pra comunidade também é importante, gostaria do seu apoio!

**Comment by Raisess** [comment]
Vamos sim ampliar o open-source, estou ansioso pelas suas PRs e vc tem todo o meu apoio.
